File: make_icon.js
const sharp = require('sharp');
const path = require('path');

const src = path.join(__dirname, 'public/icon-512.png');
const dest = path.join(__dirname, 'ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-512@2x.png');

sharp(src)
  .resize(1024, 1024)
  .flatten({ background: { r: 0, g: 0, b: 0 } })
  .toFormat('png')
  .toFile(dest)
  .then(() => console.log('Icon saved to:', dest))
  .catch(err => {
    // sharp not available, try sips
    console.log('sharp failed, trying sips...');
    const { execSync } = require('child_process');
    execSync(`sips -z 1024 1024 "${src}" --out "${dest}"`);
    console.log('Icon saved via sips');
  });

File: make_icon.py
from PIL import Image
img = Image.open('/Users/adpspare/Documents/Documents - ADP\'s MacBook Air/ALIST/a-list-2026-new/public/icon-512.png').convert('RGBA')
bg = Image.new('RGBA', (1024, 1024), (0, 0, 0, 255))
img_1024 = img.resize((1024, 1024), Image.LANCZOS)
bg.paste(img_1024, (0, 0), img_1024)
final = bg.convert('RGB')
dest = '/Users/adpspare/Documents/Documents - ADP\'s MacBook Air/ALIST/a-list-2026-new/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-512@2x.png'
final.save(dest, 'PNG')
print('Icon saved:', final.size, dest)
